pkg/storage/etcd/etcd_helper.go

```diff
@@ -580,7 +608,10 @@ func (h *etcdHelper) GuaranteedUpdate(ctx context.Context, key string, ptrToType
 		}
 
 		if string(data) == origBody {
-			return nil
+			// If we don't send an update, we simply return the currently existing
```

Comment:
No I didn't. I will add tomorrow.
@smarterclayton Are you fine with me self-applying "lgtm" label after adding a test?